src/modules/app/routes/i18nRedirect.js

```diff
@@ -24,33 +18,7 @@ export const i18nRedirectRoute = {
   path: '/:param?',
   thunk: async (dispatch: Dispatch, getState: GetState) => {
     const state = getState()
-    const param = state.location.payload.param
 
-    const citiesPayload = await citiesEndpoint.loadData(dispatch, state.cities)
-
-    if (!citiesPayload.data) {
-      // todo error handling
-    }
-
-    // the param does not exist (or is 'landing'), so redirect to the landing page with the detected language
-    if (!param || param === 'landing') {
-      dispatch(redirect(goToLanding(i18n.language)))
-      return
-    }
-
-    // the param is a valid city, so redirect to the categories route with the detected language
-    if (citiesPayload.data.find(_city => _city.code === param)) {
-      dispatch(redirect(goToCategories(param, i18n.language)))
-      return
-    }
-
-    // the param is probably a language code, so redirect to the landing route
-    if (param.length >= MIN_LANGUAGE_CODE_LENGTH || param.length <= MAX_LANGUAGE_CODE_LENGTH) {
-      dispatch(redirect(goToLanding(param)))
-      return
-    }
-
-    // param is neither a language code nor a city, so it is not a valid route
-    dispatch(redirect(goToNotFound(param)))
+    await citiesEndpoint.loadData(dispatch, state.cities, {})
```

Comment:
We don't need that any more right?

Reply:
We can leave it there for now.

Reply:
yes we need it, otherwise we would not have the cities in the I18nRedirectPage to check whether the param is a valid city